app/models/access_token.rb

```diff
@@ -1,6 +1,6 @@
 class AccessToken < ApplicationRecord
   TIMESTAMP_FORMAT = '%FT%T%:z'.freeze
-  PAST_TIME = DateTime.new(1970).utc.freeze
+  PAST_TIME = Time.at(0).utc.freeze
```

Comment:
FYI I decided to bite the bullet and risk a flame war by asking for a clarification about this time comparison stuff.

https://github.com/rubocop/rubocop/discussions/13469